samples/05_content_publishers/Identify_Items_That_Use_Insecure_URLs.ipynb

```diff
@@ -38,7 +38,7 @@
    "metadata": {},
```

Comment:
<p>Please use a profile or 'home' mode</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I pushed a pull request to @shivanip32's branch changing this to</p><p><em>gis = GIS(profile="your_online_profile")</em></p>

Reply:
<p>I pushed a second pull request changing it to</p><p><em>gis = GIS('home')</em></p>